app/ios/Podfile.lock

```diff
@@ -166,4 +174,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ef19549a9bc3046e7bb7d2fab4d021637c0c58a3
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.13.0
```

Comment:
As per from your information, the cocoapods version change was causing some issue in iOS builds. Is this resolved now?

Reply:
I think it is resolved, yes.